ssa.go

```diff
@@ -1253,16 +1262,23 @@ func (s Subtitles) WriteToSSA(o io.Writer) (err error) {
 		var b = []byte("\n[Events]\n")
 
 		// Format
-		var formatMap = make(map[string]bool)
 		var format = []string{
+			ssaEventFormatNameMarked,
 			ssaEventFormatNameStart,
 			ssaEventFormatNameEnd,
+			ssaEventFormatNameStyle,
+			ssaEventFormatNameName,
+			ssaEventFormatNameMarginL,
+			ssaEventFormatNameMarginR,
+			ssaEventFormatNameMarginV,
+			ssaEventFormatNameEffect,
+		}
+		if v4plus {
+			format[0] = ssaEventFormatNameLayer
 		}
 		var events []*ssaEvent
 		for _, i := range s.Items {
-			var e = newSSAEventFromItem(*i)
-			format = e.updateFormat(formatMap, format)
```

Comment:
Could you remove the declaration of `updateFormat()` as well?

Reply:
sure, i've change it on the last  commit.

Reply:
I don't see the removal of the declaration of `updateFormat()` line 671 in your last commit 🤔